services/configuration/src/schema.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Storage schema for the configuration service.
+
+use std::borrow::Cow;
+
+use exonum::blockchain::gen_prefix;
+use exonum::encoding::Field;
+use exonum::crypto::{self, CryptoHash, Hash, PublicKey, Signature};
+use exonum::storage::{Fork, ProofListIndex, ProofMapIndex, Snapshot, StorageValue};
```

Comment:
Technically, exonum is an external crate here and we have an agreement that such using should be before standard ones.